.gitignore

```diff
@@ -15,3 +15,4 @@ tests/Cli/Fixtures/challenges/.well-known/acme-challenge/*
 !tests/Cli/Fixtures/challenges/.well-known/acme-challenge/.gitkeep
 tests/Core/Fixtures/challenges/.well-known/acme-challenge/*
 !tests/Core/Fixtures/challenges/.well-known/acme-challenge/.gitkeep
+.idea
```

Comment:
This should be in your global gitignore (have a look at https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).